libs/lb/lb-rs/src/service/search.rs

```diff
@@ -197,7 +197,6 @@ impl Lb {
         Ok(())
     }
 
-    #[instrument(level = "debug", skip(self))]
```

Comment:
This function just spawns a thread whose task is itself instrumented. It also misleadingly results in log statements related to building indexes when `self.config.background_work` is `false`